Src/Plugins/Tests.SalModernization/TestData/UpdateSalToCurrentVersion/ExpectedOutputs/SEC105_001.AnalysisAssume.sarif

```diff
@@ -5,63 +5,79 @@
     {
       "tool": {
         "driver": {
-          "name": "testhost",
-          "organization": "Microsoft Corporation",
-          "product": "Microsoft.TestHost",
-          "fullName": "testhost 15.0.0.0",
-          "version": "15.0.0.0",
-          "semanticVersion": "15.0.0",
-          "rules": [
-            {
-              "id": "SEC105/001",
-              "name": "RemoveObsoleteOrRedundantAnnotations",
-              "fullDescription": {
-                "text": "Obsolete or redundant SAL annotation."
-              },
-              "messageStrings": {
-                "NotApplicable_InvalidMetadata": {
-                  "text": "'{0}' was not evaluated for check '{1}' because the analysis is not relevant for the following reason: {2}."
+          "name": "Spmi",
+          "organization": "Microsoft",
+          "product": "Microsoft SARIF Pattern Matcher",
+          "fullName": "Sarif.PatternMatcher 1.11.0.0",
+          "version": "1.11.0",
+          "dottedQuadFileVersion": "1.11.0.0",
```

Comment:
>dottedQuadFileVersion [](http://example.com/codeflow?start=11&length=21)

fixed bug from the SDK! we previously mishandled emitting the dotted quad file version. two args were swapped that broke proper construction. now fixed.